build/upgrade_pdfjs.sh

```diff
@@ -29,6 +29,7 @@ echo "--------------------------------------------------------------------------
 echo "Copying legacy files/folders from version ${DOC_CURRENT_ASSETS_VERSIONS}..."
 echo "-----------------------------------------------------------------------------------"
 \cp -R src/third-party/doc/${DOC_CURRENT_ASSETS_VERSIONS}/exif.js ${DOC_STATIC_ASSETS_PATH}/
```

Comment:
Do we still need this library?

Reply:
We need that for the old preloader to work. I didn't change any of the logic in the old preloader in case we have to rollback the split for whatever reason. Technically, we could get rid of the library now and use a common readEXIF function but that would require me to also modify the current non doc-first preloader class. Once the new preloader is ga'd we can get rid of the old preloader and exif library completely.